fdbclient/admin_client.go

```diff
@@ -110,6 +110,16 @@ func NewCliAdminClient(cluster *fdbv1beta2.FoundationDBCluster, _ client.Client,
 	}, nil
 }
 
+// getMaxTimeout returns the maximum timeout, this is either the default of 40 seconds or if the provided default timeout
+// is higher it will be the default cli timeout.
+func (client *cliAdminClient) getMaxTimeout() time.Duration {
+	if DefaultCLITimeout > 40*time.Second {
+		return DefaultCLITimeout
```

Comment:
It appears "DefaultCLITimeout" is getting set to "10 * time.Second" by default. Can the user change it?

Reply:
Yes, the user can use this flag: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/setup/setup.go#L90, which will be used here: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/setup/setup.go#L141